calendar_event_state/data/calendar_state_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Alessio Gerace - Agile Business Group
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+    <data noupdate="1">
+
+        <record id="event_to_plan" model="calendar.event.state">
+            <field name="name">To Plan</field>
+            <field name="sequence" eval="1"/>
+            <field name="fold" eval="True"/>
```

Comment:
@archetipo I would set 'to plan' and 'planned' states with fold = False by default. What do you think?